### ISSUE TITLE
Allow to save GPX as Simplified version for other apps

### ISSUE THREAD

**Issue by joeldebruijn** [osmandapp/web#436]
**Background:**

- I prepare trips in OSMand Web
- After finding the right route, its exported as a GPX file.
- I save them locally for longterm archival and sometime view them in other apps (like Gnome Maps or Organic Maps).

**Symptom:**
When opening a GPX file (made by OSMand) in another app it shows a straight returning line.
![afbeelding](https://github.com/user-attachments/assets/b102ed20-08a1-443e-a145-f4158e8fb81f)

I tested it in 2 other apps and it shows. 
Imported in OSMand Android this line is absent though.

**Questions:**

- Is this an error of the other apps by showing a return line?
- Is there something wrong in the produced GPX files?

**Comment by EugeneZmeuk** [comment]
Hi @joeldebruijn 

I created the GPX file on the web [Sat Jul 20 2024 - 2.gpx.zip](https://github.com/user-attachments/files/16319412/Sat.Jul.20.2024.-.2.gpx.zip) : 

<img width="859" alt="image" src="https://github.com/user-attachments/assets/641829d7-0b91-4401-bc1b-c2827228f7a7">

After I opened this file by OsmAnd:

<img width="371" alt="image" src="https://github.com/user-attachments/assets/16fc42ee-dba3-4b2a-b501-f4a3d32b54a7">

**This file has two types of tags inside** :
1. <RTE>
 <details>
  <summary><rte> - route segment: it's a straight lines on your example:</summary>


```
  <rte>
    <rtept lat="51.4595155" lon="19.7813988">
      <extensions>
        <osmand:profile>bicycle</osmand:profile>
        <osmand:trkpt_idx>0</osmand:trkpt_idx>
      </extensions>
    </rtept>
    <rtept lat="51.4452346" lon="19.8312664">
      <extensions>
        <osmand:profile>bicycle</osmand:profile>
        <osmand:trkpt_idx>43</osmand:trkpt_idx>
      </extensions>
    </rtept>
    <rtept lat="51.430093" lon="19.8674011">
      <extensions>
        <osmand:profile>bicycle</osmand:profile>
        <osmand:trkpt_idx>55</osmand:trkpt_idx>
      </extensions>
    </rtept>
    <rtept lat="51.4104495" lon="19.8491192">
      <extensions>
        <osmand:profile>bicycle</osmand:profile>
        <osmand:trkpt_idx>75</osmand:trkpt_idx>
      </extensions>
    </rtept>
    <rtept lat="51.3934756" lon="19.8487759">
      <extensions>
        <osmand:profile>bicycle</osmand:profile>
        <osmand:trkpt_idx>144</osmand:trkpt_idx>
      </extensions>
    </rtept>
    <rtept lat="51.3726377" lon="19.8429394">
      <extensions>
        <osmand:profile>bicycle</osmand:profile>
        <osmand:trkpt_idx>282</osmand:trkpt_idx>
      </extensions>
    </rtept>
    <rtept lat="51.352916" lon="19.8306656">
      <extensions>
        <osmand:profile>bicycle</osmand:profile>
        <osmand:trkpt_idx>344</osmand:trkpt_idx>
      </extensions>
    </rtept>
  </rte>
```
</details>


2. classic point-to-point  <TRK>

 <details>
  <summary><trk> - with point segment <trkseg> with coordinats and elevation in our case:</summary>


```
 <trk>
    <trkseg>
      <trkpt lat="51.459559" lon="19.7814786">
        <ele>201</ele>
      </trkpt>
      <trkpt lat="51.4591662" lon="19.7820264">
        <ele>201</ele>
      </trkpt>
      <trkpt lat="51.4591428" lon="19.7820586">
        <ele>199.7</ele>
      </trkpt>
      <trkpt lat="51.4591161" lon="19.7820157">
        <ele>201</ele>
      </trkpt>
      <trkpt lat="51.4587768" lon="19.7824985">
        <ele>200.2</ele>
      </trkpt>
      <trkpt lat="51.4558053" lon="19.7865567">
        <ele>201.8</ele>
      </trkpt>
      <trkpt lat="51.455476" lon="19.7869912">
        <ele>201</ele>
      </trkpt>
      ............
```
</details>

The Komoot app can open one or second variants:

| choosing of variants | **<trk>** |
|----|----|
|![telegram-cloud-photo-size-2-5249043244166275625-y](https://github.com/user-attachments/assets/55863a4d-db7b-44ee-ad7f-236c611566c8)|![telegram-cloud-photo-size-2-5249043244166275626-y](https://github.com/user-attachments/assets/822e7f9e-66ff-486e-8e3b-a78b6d4643fb)|

But the Organics maps app opens as you wrote with a mix:
<img width="374" alt="image" src="https://github.com/user-attachments/assets/79cc32a7-13ea-47e3-b06d-d1baa88f7075">

**Comment by joeldebruijn** [comment]
Oh thnx for reproducing and sharing this, never new GPX could have these 2 "types of Tour". 🙏

Is there a way for OSMand to generate with only point-to-point?

Otherwise I make an issue for Organic Maps to treat GPX import the way Komoot does.

**Comment by EugeneZmeuk** [comment]
> Oh thnx for reproducing and sharing this, never new GPX could have these 2 "types of Tour". 🙏
> 
> Is there a way for OSMand to generate with only point-to-point and without route segment?
> 
> Otherwise I make an issue for Organic Maps to treat GPX import the way Komoot does.

While you can do it on the OsmAnd App
Open track menu > Plan a route button (Edit track) > Opitons > Save as new track > choose the "Simplified Track" option and save it.
and only after that this track will be have only point-to-point 

https://osmand.net/docs/user/plan-route/create-route/#save-as-new-track

**Comment by joeldebruijn** [comment]
Your explanation helped me looking into the GPX standard, learned about rtept en trkpt differences.
eg https://gis.stackexchange.com/questions/403664/how-route-points-rtept-and-track-points-trkpt-can-exist-in-a-gpx-file-togeth

I was inclined to close this issue and make a new one, asking for the "Simplefied Track" option for OSM web.
Preferably somewhere here:
![afbeelding](https://github.com/user-attachments/assets/67014d2a-0161-4238-bb06-0e4cf0386b89)

But I dont know if OSMand web is meant to have the same features as OSMand Android/iOS?
Because then the backlog for web becomes huge I guess?

**Comment by EugeneZmeuk** [comment]
@joeldebruijn 
Yes, it's a good decision to make a new issue. Please, create it here.
yesterday we had a short discussion about this case and the "Simplified track" is needed for the Options list.

**Comment by alisa911** [comment]
[Figma](https://www.figma.com/design/fqiiJCrepmJP7XkA3YixSq/My-Places?node-id=86-16338&m=dev)

**Comment by alisa911** [comment]
<img width="2169" height="1012" alt="Image" src="https://github.com/user-attachments/assets/ec3dc16e-d5a9-4c90-86e1-dc66826411b3" />
<img width="2129" height="1116" alt="Image" src="https://github.com/user-attachments/assets/9d4c44c8-3d38-4fe9-b121-cfaaef43ff13" />
<img width="2223" height="1025" alt="Image" src="https://github.com/user-attachments/assets/e01790b6-32a0-4ec5-999a-a38a2d56104e" />
